packages/components/psammead-most-read/src/index.stories.jsx

```diff
@@ -7,31 +7,41 @@ import Timestamp from '@bbc/psammead-timestamp';
 import MostReadList from './List';
 import MostReadTitle from './Title';
 import { MostReadLink, MostReadRank } from './Item';
-import { items, itemsRTL } from './testHelpers/fixtureData';
 import MostRead from './index';
 
 const arabicServiceDecorator = withServicesKnob({
   defaultService: 'arabic',
 });
 
-const bengaliServiceDecorator = withServicesKnob({
-  defaultService: 'bengali',
+const newsServiceDecorator = withServicesKnob({
+  defaultService: 'news',
 });
 
-const burmeseServiceDecorator = withServicesKnob({
-  defaultService: 'burmese',
+const BASE_PATH = 'https://www.bbc.com';
+const getItem = (articlePath, text) => ({
```

Comment:
I feel like we could just re-use this function: packages/components/psammead-most-read/src/testHelpers/itemsHelper.js